packages/fxa-auth-server/test/local/routes/validators.js

```diff
@@ -386,17 +411,48 @@ describe('lib/routes/validators:', () => {
     });
 
     it('accepts unexpected keys', () => {
-      const res = subject.validate({
-        'capabilities:8675309': '123done,321done',
-        newThing: 'this is unexpected',
-      });
+      const res = subject.validate(
+        Object.assign({}, validProductMetadata, {
+          'capabilities:8675309': '123done,321done',
+          newThing: 'this is unexpected',
+        })
+      );
       assert.ok(!res.error);
     });
 
     it('rejects expected keys with invalid values', () => {
-      const res = subject.validate({
-        iconURL: true,
-      });
+      const res = subject.validate(
+        Object.assign({}, validProductMetadata, {
+          webIconURL: true,
+        })
+      );
```

Comment:
This is only checking for an invalid `webIconURL`.  We have validation rules for the other values as well, so I'd expect to see more here.

Reply:
These are all tested above in `subscriptionProductMetadataValidator`

Reply:
Huh aren't we in `subscriptionProductMetadataValidator` here (line 375)?  Not sure I understand.  Can you link to or give me the line number then?

Reply:
Cool, thanks.  An extra thing we could do here, now that we ensure that those keys are being validated, is to test for more than a type mismatch to trigger the validation error, especially for something that we defined, like the regex for the download urls.  The `isValidEmailAddress` and `redirectTo` tests in this file are good examples.